app/models/tools/saml_request.rb

```diff
@@ -46,12 +46,19 @@ def check_signature_validity
       @valid_signature ||= begin
 
         if certs.nil?
-          @errors.push('Could not find any certificates to use. Please add a certificate to your application configuration or paste one below.')
+          @errors.push(
+            'Could not find any certificates to use. Please add a ' +
+            'certificate to your application configuration or paste one below.',
+          )
```

Comment:
what about heredoc syntax here?
```suggestion
          @errors.push(<<~EOS.squish)
            Could not find any certificates to use. Please add a
            certificate to your application configuration or paste one below.
          EOS
```

Reply:
updated in https://github.com/18F/identity-dashboard/pull/776/commits/8726d8eefc76b44db9fdc5eb9c75e8ae25dba821